src/content/blog/astro-510.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "Astro 5.1"
+description: "Astro 5.1 is now available. This includes a new experimental sessions feature, improved caching for images, and new tools for building Astro Actions."
+homepageLink:
+  title: "Astro 5.1"
+  subtitle: "Available now!"
+publishDate: "2024-12-17"
+authors:
+  - matt
+coverImage: "/src/content/blog/_images/astro-510/blog-post-5-1.webp"
+socialImage: "/src/content/blog/_images/astro-510/og-astro-5-1.webp"
+lang: "en"
+---
+
+import BlogContentImage from "/src/components/BlogContentImage.astro";
+
+**Astro 5.1 is now available! Delivered just in time for the holidays, the first minor release of the Astro 5 era includes a new experimental sessions feature, improved caching for images, and extra tools for building Astro Actions.**
+
+The gifts it bears include:
+
+- [**Remote image caching**](#remote-image-caching)
+- [**Experimental: Sessions**](#experimental-sessions)
```

Comment:
I would move experimental sessions first, since it's the biggest feature of the release